src/platform_impl/apple/appkit/view.rs

```diff
@@ -777,13 +778,20 @@ define_class!(
             trace_scope!("acceptsFirstMouse:");
             self.ivars().accepts_first_mouse
         }
+
+        #[unsafe(method(shouldDelayWindowOrderingForEvent:))]
+        fn should_delay_window_ordering_for_event(&self, _event: &NSEvent) -> bool {
+            trace_scope!("shouldDelayWindowOrderingForEvent:");
+            !self.focusable()
+        }
```

Comment:
Nit: To support all the functionality in `shouldDelayWindowOrderingForEvent:`, we'd probably need to send an event synchronously here, and let users enable/disable `focusable` before the method finishes. That can be done later though, but could you add a comment about that here for now?